src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -50,25 +29,32 @@ public Book create(@RequestBody Book book ) {
         return bookRepository.save(book);
     }
 
-    @GetMapping("/title/{bookTitle}")
-    public Optional<Book> findByTitle(@PathVariable String bookTitle) {
-        return bookRepository.findByTitle(bookTitle);
+    @GetMapping("/view")
+    public Iterable findAll() {
+        return bookRepository.findAll();
+    }
+
+    @GetMapping("/")
+    public Iterable<Book> findBooks(@RequestParam(name="bookTitle", required=false, defaultValue="") String bookTitle,
+                                    @RequestParam(name="author", required=false, defaultValue="") String author,
+                                    @RequestParam(name="isbn", required=false, defaultValue="") String isbn) {
+        System.out.println(bookTitle);
+        return bookRepository.findByAuthorContainingAndTitleContainingAndIsbnContainingAllIgnoreCase( author, bookTitle, isbn);
     }
 
-    @GetMapping("/{id}")
-    public Optional<Book> findOne(@PathVariable Long id) {
+    @GetMapping("/view/{id}")
```

Comment:
The /view part is incorrect, REST endpoints for particular resources are directly /{id}